src/robot_manager.cpp

```diff
@@ -445,16 +445,30 @@ try
   if (loaded_robot->isLocal())
   {
     TEMOTO_DEBUG_STREAM_("Creating a manipulation path for robot '" << loaded_robot->getName() 
-      << " with goal pose: " << req.target_pose <<std::endl);
+      << " with goal pose: " << req.goal_target <<std::endl);
 
-    if (req.use_named_target)
+    temoto_robot_manager::RobotPlanManipulation rpm_msg;
+
+    switch (req.goal_target)
     {
-      loaded_robot->planManipulationPath(req.planning_group, req.named_target);
+      case rpm_msg.request.DEFAULT_TARGET:
```

Comment:
instead of `rpm_msg.request.DEFAULT_TARGET`, do `RobotPlanManipulation::Request::DEFAULT_TARGET`. while it's longer, it's more clear that it is a static property of the `RobotPlanManipulation` object

Reply:
got it